dom/examples/todo/src/main_section.rs

```diff
@@ -35,8 +35,8 @@ pub fn toggle(default_checked: bool) -> Span {
 #[illicit::from_env(todos: &Key<Vec<Todo>>, visibility: &Key<Visibility>)]
 pub fn todo_list() -> Ul {
```

Comment:
See my comments in other todomvc files, I think if you add `Commit<Visibility>` and `Commit<Vec<Todo>>` to the illicit environment then you can avoid all the commit_at_root calls in this file.